package.json

```diff
@@ -49,15 +49,16 @@
     "compodoc": "compodoc -w -p tsconfig.json -s -n \"huddly-sdk\" -d \"docs\" --minimal --customFavicon \"docs_style/assets/imgs/favicon.ico\" --disableGraph --disableInternal --hideGenerator --includesName \"Additional Information\"  --includes examples -a docs_style/assets -y docs_style",
     "prepare": "npm run build",
     "preversion": "npm run tslint",
-    "prettier": "./node_modules/.bin/prettier  --config ./.prettierrc --write 'src/**/*.ts'",
-    "prettier-check": "./node_modules/.bin/prettier  --config ./.prettierrc --check 'src/**/*.ts'",
+    "prettier": "./node_modules/.bin/prettier  --config ./.prettierrc --write \"src/**/*.ts\"",
+    "prettier-check": "./node_modules/.bin/prettier  --config ./.prettierrc --check \"src/**/*.ts\"",
+    "prettify-staged": "./node_modules/.bin/pretty-quick --staged --pattern \"src/**/*.ts\" --verbose",
     "release:major": "./node_modules/generate-changelog/bin/generate -M && git add CHANGELOG.md && git commit -m 'Updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
     "release:minor": "./node_modules/generate-changelog/bin/generate -m && git add CHANGELOG.md && git commit -m 'Updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
     "release:patch": "./node_modules/generate-changelog/bin/generate -p && git add CHANGELOG.md && git commit -m 'Updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
```

Comment:
Remove the `./node_modules/` prefix on the commands as that wont work on some shells:

```
'.' is not recognized as an internal or external command
```

Reply:
You want me to do this for all the scripts starting with ./node_modules?

Reply:
Yes please.